### PR TITLE
More stage boards and info, new species

### PR DESCRIPTION
**Pull request by Manitary**
Stage 276 (Rattata) default board reworked
Stage SP_503->SP_503T (Zygarde-10) renamed to match pokemon name
Stage SP_339T (Landorus-T) default board added
New species added: Ash-Greninja, Landorus-T, Zygarde 50%
Stages info updated: escalation battle HP added to Latias, Keldeo-R, Zygarde 50%

### CODE REVIEW

File: resources/config/defaults/boards/276.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ STAGE 276
 ROW_1 Air,Wood,Air,Wood,Air,Wood
 FROW_1 false,false,false,false,false,false
 ROW_2 Air,Wood,Air,Wood,Air,Wood
-FROW_2 false,false,false,false,false,false
+FROW_3 false,false,false,false,false,false
```
Comment (Loreinator) [inline resources/config/defaults/boards/276.txt]: Why? This makes no sense

Comment (Manitary) [inline resources/config/defaults/boards/276.txt]: Of course I had to mess up with the revision itself, now it should be correct.

(Next time I'll be more clean, I just learned about squashing changes into a single commit)

Comment (Loreinator) [inline resources/config/defaults/boards/276.txt]: Don't worry about it, I've copied the verified parts from your pull request into what I was already working on. I'll be pushing it in either tonight or tomorrow and begin the testing phase (and hopefully the deployment phase shortly after).

**Comment by Loreinator** [comment]
Thank you for your assistance, changes have been included in the following commits:

Species and stage data added: acfc45f6ff257a55012919f497fb420027d44c38

Board corrected: b35ae0b3e9e5dfc359febb7777d2510cc411e713

The stage name for SP_503 was already SP_503T.

**Pull request closed without merging**